test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/BulkUpdateTests.cs

```diff
@@ -492,67 +492,51 @@ public async Task GivenBulkUpdateJobWithMoreThanOnePageOfIncludeResultsWithDiffe
             patchRequest = new Parameters()
                 .AddAddPatchParameter("Group", "active", new FhirBoolean(true));
+
             ChangeTypeToUpsertPatchParameter(patchRequest);
-            queryParam = new Dictionary<string, string>
-                {
-                    { "_include", "Group:member" },
-                    { "_isParallel", isParallel.ToString() },
-                };
-            using HttpResponseMessage responseIgnored = await SendBulkUpdateRequest(tag, patchRequest, "Group/$bulk-update", queryParam);
+            using HttpResponseMessage responseIgnored = await SendBulkUpdateRequest(tag, patchRequest, "Patient/$bulk-update", new Dictionary<string, string>() { { "_isParallel", isParallel.ToString() } });
             Assert.Equal(HttpStatusCode.Accepted, responseIgnored.StatusCode);
 
             BulkUpdateResult expectedResultsForIgnored = new BulkUpdateResult();
             expectedResultsForIgnored.ResourcesIgnored.Add("Patient", 2005);
-            expectedResultsForIgnored.ResourcesUpdated.Add("Group", 1);
             await MonitorBulkUpdateJob(responseIgnored.Content.Headers.ContentLocation, expectedResultsForIgnored);
 
             // For Patch failures
             patchRequest = new Parameters()
                 .AddAddPatchParameter("Patient", "id", new FhirString("newId"))
                 .AddAddPatchParameter("Group", "active", new FhirBoolean(true));
+
             ChangeTypeToUpsertPatchParameter(patchRequest);
-            queryParam = new Dictionary<string, string>
-                {
-                    { "_include", "Group:member" },
-                    { "_isParallel", isParallel.ToString() },
-                };
-            using HttpResponseMessage responsePatchFailed = await SendBulkUpdateRequest(tag, patchRequest, "Group/$bulk-update", queryParam);
+
+            using HttpResponseMessage responsePatchFailed = await SendBulkUpdateRequest(tag, patchRequest, "Patient/$bulk-update", new Dictionary<string, string>() { { "_isParallel", isParallel.ToString() } });
             Assert.Equal(HttpStatusCode.Accepted, responsePatchFailed.StatusCode);
 
             BulkUpdateResult expectedResultsPatchFailed = new BulkUpdateResult();
             expectedResultsPatchFailed.ResourcesPatchFailed.Add("Patient", 2005);
-            expectedResultsPatchFailed.ResourcesUpdated.Add("Group", 1);
             await MonitorBulkUpdateJob(responsePatchFailed.Content.Headers.ContentLocation, expectedResultsPatchFailed);
         }
 
         [SkippableTheory]
         [InlineData(true)]
         [InlineData(false)]
         [HttpIntegrationFixtureArgumentSets(DataStore.SqlServer, Format.Json)]
-        public async Task GivenBulkUpdateJobWithMoreThanOnePageOfResultsWithDifferentScenarios_WhenCompleted_ThenResultsAreMatched(bool isParallel)
+        public async Task GivenBulkUpdateJobWithMoreThanOnePageOfIncludeResultsWithDifferentScenarios_WhenCompleted_ThenResultsAreMatched(bool isParallel)
```

Comment:
This test looks like it has four tests inside of it: normal, ignored, failed, and a more complex include. Should these all be in one test or split up?